model/communication/Endpoint.ttl

```diff
@@ -99,30 +86,6 @@ ids:outboundPath
     rdfs:comment "Relative path used for outbound communication via this endpoint, i.e., output messages and faults only."@en.
 
 
-ids:appEndpointType
-    a owl:ObjectProperty ;
-    rdfs:domain ids:AppEndpoint ;
-    rdfs:range ids:AppEndpointType ;
-    rdfs:label "App endpoint type"@en ;
-    rdfs:comment "Type of an app endpoint. Data apps have different endpoint types, e.g., for input, output or health status."@en ;
-.
-
-ids:appEndpointPort
```

Comment:
Am I getting it right that this and other properties will be deleted? If so, please document it in the change log. I think it would make sense to draft these parts of the changelog now already, in a section "next release", as we do not yet know the version number.